packages/webpack-configs/src/build.ts

```diff
@@ -153,37 +145,7 @@ export function defineBuildConfig(swcConfig: SwcConfig, options: DefineBuildConf
             publicPath,
             clean: true
         },
-        cache: cache && {
-            type: "filesystem",
-            allowCollectingMemory: false,
-            cacheDirectory: cacheDirectory,
-            maxMemoryGenerations: 1,
-            // Took from https://webpack.js.org/configuration/cache/#cachebuilddependencies.
-            buildDependencies: {
-                config: [__filename]
-            }
-        },
-        // (ACTUALLY NOT FIXING ANYTHING AT THE MOMENT)
-        // Fixes caching for environmental variables using the DefinePlugin by forcing
-        // webpack caching to prioritize hashes over timestamps.
-        snapshot: cache ? {
-            buildDependencies: {
-                hash: true,
-                timestamp: true
-            },
-            module: {
-                hash: true,
-                timestamp: true
-            },
-            resolve: {
-                hash: true,
-                timestamp: true
-            },
-            resolveBuildDependencies: {
-                hash: true,
-                timestamp: true
-            }
-        } : undefined,
+        cache: false,
```

Comment:
According to webpack documentation, cache is disabled by default in production so you shouldn't need to set the option to `false`: https://reactrouter.com/en/main

Reply:
I'll remove this line